### DISCUSSION TITLE
Unable to upgrade past Fief 0.25.2 due to SSL error: certificate verify failed with DigitalOcean managed DB

### DISCUSSION THREAD

**Discussion by matteius** [fief-dev/fief#284] [Q&A] [answered]
Unable to upgrade past Fief 0.25.2 due to SSL error: certificate verify failed with DigitalOcean managed DB

Issue starts with v0.25.3 getting this when trying to start the deployment:

OperationalError: (psycopg2.OperationalError) connection to server at 
"db-postgresql-redacted.db.ondigitalocean.com" 
(157.230.12.240), port 25060 failed: SSL error: certificate verify failed
(Background on this error at: https://sqlalche.me/e/20/e3q8)
2023-10-07 18:43:42,932 WARN exited: fief-server (exit status 1; not expected)

Any ideas?

**Comment by matteius** [comment]
Changing `DATABASE_SSL_MODE: "disable"` allowed me to upgrade -- now I am just trying to figure out the email template sending for the verification codes.

**Comment by frankie567** [comment, marked as answer]
Hi @matteius 👋 

To create the email templates, you'll need to run the following command:

```sh
fief workspaces init-email-templates
```

---

As mentioned [here](https://github.com/fief-dev/fief/releases/tag/v0.27.0), if you want to upgrade to 0.27.0, be sure to apply 0.26.3 before.
